src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -21,6 +21,10 @@ namespace InferenceEngine {
 
 static constexpr size_t invalid_data_idx = std::numeric_limits<size_t>::max();
 
+static bool DoNotSkip(CNNLayerPtr layer) {
```

Comment:
Why don't use `CNNNetHasPrevLayer` instead of `CNNNetPrevLayerSkipCertain` with false condition?

Reply:
I wanted to simplify the client code. But ok - reverted.